<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardDetection</class>
 <widget class="QWidget" name="BoardDetection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BoardDetection</string>
  </property>
  <widget class="QPushButton" name="openCamera">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>250</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>900</width>
     <height>750</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>系统图形显示区域</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="removeShape">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>移除形状</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extractColor">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提取颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takePhoto">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>250</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>拍摄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePicture">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>250</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存图像</string>
   </property>
  </widget>
  <widget class="QSlider" name="scaleSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="scaleUp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scaleDown">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fillArea">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>填充窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fitArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>适合窗口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openPicture">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calibration">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>标定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cutPicture">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>剪裁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="measure">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>测量</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorResult">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>580</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>颜色结果</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>比对颜色</string>
   </property>
  </widget>
  <widget class="QSlider" name="cutBlurredSlider">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="cutThresholdSlider">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>90</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="cutBlurredLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CutBlurSize：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="cutThresholdLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CutBinThresh：100</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratioLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ratio：0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixelFatorLabel">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PixelFator：0.000000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undistort">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>矫正畸变</string>
   </property>
  </widget>
  <widget class="QSlider" name="meaThresholdSlider">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>210</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="meaBlurredSlider">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>150</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="meaBlurredLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MeasureBlurSize：50</string>
   </property>
  </widget>
  <widget class="QLabel" name="meaThresholdLabel">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MeasureBinThresh：100</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdits">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>390</y>
     <width>251</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>openCamera</tabstop>
  <tabstop>calibration</tabstop>
  <tabstop>measure</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
